doc/classes/EditorSettings.xml

```diff
@@ -722,6 +722,9 @@
 			Specify the multiplier to apply to the scale for the editor gizmo handles to improve usability on touchscreen devices.
 			[b]Note:[/b] Defaults to [code]1[/code] on non-touchscreen devices.
 		</member>
+		<member name="network/connection/network_mode" type="int" setter="" getter="">
+			Determines whether online features are enabled in the editor, such as the Asset Library. Setting this property to "Offline" is recommended to limit editor's internet activity, especially if privacy is a concern.
```

Comment:
Is it disabling export template download as well? If it's not, it's probably should be named more specific, like `network/connection/online_asset_library`. Since export template dialog will download mirror list without asking, so it's not "offline".

Reply:
Should probably be extended to cover that part too, or a separate setting

Reply:
I addressed this in the OP:

> The same should probably apply to the dialog which allows you to download export templates, and other editor tools with online activity, but pardon me, I haven't implemented it in this PR for simplicity's sake.

It is a generic setting, and everything that wants to go online in the editor must respect it. I just didn't go through everything to make it so for this PR.